darc/mock_resolver_outputs/test_resolver

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
These files will go away, the content can be inside of the unit tests directly. (less files = more good, within reason)